src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big {
+  padding-right: 62px;
+}
+
+.small {
+  padding-right: 33px;
+}
+
+.input {
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  border-radius: 4px;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
```

Comment:
It's a too small change, it's strange that it is in figma, but usually we don't change border-radius to 1px on state change. I think it can be removed